src/solvers/refinement/string_constraint_instantiation.cpp

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************\
+
+Module: Defines functions related to string constraints.
+
+Author: Jesse Sigal, jesse.sigal@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Defines related function for string constraints.
+
+#include <solvers/refinement/string_constraint_instantiation.h>
+
+#include <solvers/refinement/string_constraint.h>
+#include <solvers/refinement/string_constraint_generator.h>
+#include <solvers/refinement/string_refinement.h>
+
+/// Instantiates a quantified formula representing `not_contains` by
+/// substituting the quantifiers and generating axioms.
+/// \related string_refinementt
+/// \param [in] axiom: the axiom to instantiate
+/// \param [in] index_set0: the index set for `axiom.s0()`
+/// \param [in] index_set1: the index set for `axiom.s1()`
+/// \param [in] generator: generator to be used to get `axiom`'s witness
+/// \return the lemmas produced through instantiation
+std::list<exprt> instantiate_not_contains(
```

Comment:
Since this function is only used in this file... Replace `std::list<exprt>` with `std::vector<exprt>`.
Other than that - LGTM.